frontend/src/components/LineChart/LineChart.js

```diff
@@ -6,18 +6,25 @@ import Box from '@mui/material/Box'
 import PropTypes from 'prop-types'
 
 import { useVictoryTheme, useClientSize } from 'hooks/victory'
-import { formatTickNumber, getStackData, STACK_COLORS, angledProperty } from 'utils/victory'
+import { formatTickNumber, getStackData, STACK_COLORS, angledProperty, getSettingYear } from 'utils/victory'
 import { ChartAxisType } from 'utils/propTypes'
 import ChartFlyOut from 'components/ChartFlyOut'
 
-const LineChart = ({ data, title, xAxis, yAxis }) => {
+const LineChart = ({ data, title, xAxis, yAxis, variant }) => {
   const intl = useIntl()
   const theme = useTheme()
-  const stackData = useMemo(() => getStackData(data), [data])
+  const modifiedDataWithDate = data.map(item => {
+    if (variant === 'time_years') {
+      return { ...item, x: new Date(new Date(item.x).setFullYear(getSettingYear(2022, item.x))) }
+    } else if (variant === 'time') {
+      return { ...item, x: new Date(item.x) }
+    }
+    return item
+  })
+  const stackData = useMemo(() => getStackData(modifiedDataWithDate), [modifiedDataWithDate])
```

Comment:
Whatever we do here should done the same way in `<StackChart>` as well. Probably best to pass `variant` into `getStackData()` and simplify this part of `<LineChart>` and `<StackChart>` down to a single call to `getStackData()`.